### DISCUSSION TITLE
BACKUP GCS interval

### DISCUSSION THREAD

**Discussion by akshayparseja** [Altinity/clickhouse-backup#1020] [Q&A] [answered]
I want to setup backup interval to gcs on weekly basis. I dont see a configuration around it
I see some FULL interval but that is when watch command runs
i am running this image as a container which backedup data to gcs but not sure how to configure it for weely basis

**Comment by Slach** [comment, marked as answer]
You can try to use `watch` command where `full_interval` config parameter applicable, 
but better just use weekly `cron` if you use standalone server
if you use just docker image you can try something like that
`0 5 * * 1 docker run --rm -v /var/lib/clickhouse:/var/lib/clickhouse altinity/clickhouse-backup:latest -- clickhouse-backup create_remote backup-$(date +%Y%m%d)`


moreover, please read
https://github.com/Altinity/clickhouse-backup/#dont-run-clickhouse-backup-remotely
